rosplan_demos/common/deterministic_domain_turtlebot.pddl

```diff
@@ -0,0 +1,53 @@
+(define (domain turtlebot)
+
```

Comment:
It would be good to rename the existing turtlebot domain as well. Perhaps:
temporal_domain_turtlebot.pddl